xbmc/guilib/GraphicContext.cpp

```diff
@@ -369,19 +369,19 @@ bool CGraphicContext::IsValidResolution(RESOLUTION res)
 }
 
 // call SetVideoResolutionInternal and ensure its done from mainthread
-void CGraphicContext::SetVideoResolution(RESOLUTION res, bool forceUpdate)
+void CGraphicContext::SetVideoResolution(RESOLUTION res, bool forceUpdate, bool callWindowing)
```

Comment:
we should route all events via the same path. if the compositor wants a different resolition we should handle this the same way as we handle i.e. keypresses